### ISSUE TITLE
Publish docker images

### ISSUE THREAD

**Issue by glensc** [perftools/xhgui#311]
We should publish pre-built docker images, so that setting up an instance is faster. as currently, the build process takes some time. and save green energy! 👍 

```
➔ time docker build --no-cache .
real	2m8.080s
user	0m1.258s
sys	0m1.331s
```

This kind of also ensures the builds are verified to be working the same for everybody (we don't track `composer.lock`, so the outcome may vary depending on when you build).

In the future, we could build separate mongo and pdo variants.

**Comment by glensc** [comment]
@markstory can you organize access please:

create org on hub.docker.com and add user glen access to push.
or you prefer to use github infra (they have also docker registry support)?

I've personally have not used github infra or dockerhub automation, so I'd go with manual pushing to dockerhub.

**Comment by glensc** [comment]
# Re: variants

I don't think we need multiple variants, rather add support for both modules. it's just PHP extensions.

**Comment by markstory** [comment]
@glensc I created the dockerhub org and added you as an owner. :smile:

**Comment by glensc** [comment]
published for the last tag (0.13.0):
- https://hub.docker.com/repository/docker/xhgui/xhgui

**Comment by glensc** [comment]
PR adding more pdo drivers: https://github.com/perftools/xhgui/pull/323
